assets/js/components/ChecksCatalog/ChecksCatalogNew.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+
+import axios from 'axios';
+
+import CatalogContainer from './CatalogContainer';
+
+const wandaURL = process.env.WANDA_URL;
+
+export const ChecksCatalogNew = () => {
+  const [catalogError, setError] = useState(null);
+  const [loading, setLoaded] = useState(true);
+  const [catalogData, setCatalog] = useState([]);
+
+  useEffect(() => {
+    getCatalog();
+  }, []);
+
+  const getCatalog = () => {
+    setLoaded(true);
+    axios
+      .get(`${wandaURL}/api/checks/catalog`)
+      .then((catalog) => {
+        setLoaded(false);
+        setCatalog(catalog.data);
+      })
+      .catch(function (error) {
+        setLoaded(false);
+        setError(error.message);
+      });
+  };
```

Comment:
You could just use an arrow function here `(error) => {setError(error.message)}`